site/content/en/docs/concepts/cluster_queue.md

```diff
@@ -429,6 +429,58 @@ Workloads as possible, preferring Workloads with these characteristics:
 - Workloads with the lowest priority.
 - Workloads that have been admitted more recently.
 
+### Preemption Algorithm
+
+The preemption algorithm has the following stages.
+
+#### Eligibility
+
+A workload is eligible for preemptions if it is eligible for preemptions for the
```

Comment:
`A workload is eligible for preemptions if it is eligible for preemptions` sounds strange.

Reply:
Right, I have rephrased the description. PTAL.